TombEngine/Objects/Generic/puzzles_keys.h

```diff
@@ -3,6 +3,13 @@
 struct CollisionInfo;
 struct ItemInfo;
 
+enum PuzzleHoleType
```

Comment:
should this be KeyHoleType?

Reply:
this enum is also used by puzzle holes. how should I call it?

Reply:
fixed